### ISSUE TITLE
How to create two related pie charts, one inside each other?

### ISSUE THREAD

**Issue by Macacoazul01** [imaNNeo/fl_chart#1557]
I need to create something like this:

![image](https://github.com/imaNNeo/fl_chart/assets/57464965/81f57670-f6f9-4141-b2ef-61e99bd72548)

Where both pie charts should be filtered if i filter one. Also the outside one should follow the inside group (as in the image).

Is this possible with the package?

**Comment by polonski** [comment]
One way is to use a Stack of Containers:

```
 return Stack(
      children: <Widget>[
        Container(
          transformAlignment: AlignmentDirectional.center,
          height: 300,
          width: 300,
          child: PieChart( ..
            PieChartData( sections:[
               PieChartSectionData(
                  radius: 50,
                  ..

        Container(
          transformAlignment: AlignmentDirectional.center,
          height: 300,
          width: 300,
          child: PieChart( ..
            PieChartData( sections:[
               PieChartSectionData(
                  radius: 100,
                  ..

  ..
      

```
If you align them in the center, each of your charts with a unique radius in its own `Container` at the same center origin.

**Comment by imaNNeo** [comment]
Yes, I think the best solution is to use a Stack widget.
btw, I will keep it open to think about it later.
Maybe it would be better if we implemented it internally. I'm not sure!

**Comment by Macacoazul01** [comment]
Is there any chances that stack breaks during resize of the web navigator (i'm developing a flutter web project)

**Comment by Macacoazul01** [comment]
Also how can I "attach" the two charts behavior?

**Comment by imaNNeo** [comment]
Interactions will not merge, unfortunately.
But there is no problem using Stack  in the web

**Comment by Macacoazul01** [comment]
Is it possible to trigger an interaction on the second chart after one on the first and vice versa?

**Comment by polonski** [comment]
> Is it possible to trigger an interaction on the second chart after one on the first and vice versa?

Yes that should be fine, it depends where you keep your state. it works well with BloC implementation

**Comment by Macacoazul01** [comment]
@imaNNeo are there any chances of this being implemented as a default chart on the package?

**Comment by imaNNeo** [comment]
> @imaNNeo are there any chances of this being implemented as a default chart on the package?

Yes, we can do this. But unfortunately, I have limited time and I don't have enough funds to work full-time here.
So I do my best to implement it asap. 
Also, contributions are welcome!
